Digitalbum/Digitalbum/Screens/AuthenticationFlow/SignUpScreen.swift

```diff
@@ -22,6 +25,11 @@ struct SignUpScreen: View {
             VStack {
                 EmailTextField(email: $email)
                 PasswordTextField(password: $password)
+                UserInfoTextField(text: $userName, placeholder: "Tell us your name!")
+                ZStack {
+                    BirthdayTextField(selectedDate: $userDateOfBirth)
+                } .zIndex(1)
```

Comment:
I suggest a simpler implementation for the DatePicker, using a DisclosureGroup. It's still up to you, if you need any help hit me up.